ballerina/tests/tests.bal

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+  
+import ballerina/test;
+import ballerina/mime; 
+
+configurable boolean isLiveServer = ?;
+configurable string openAIKey = ?;
+configurable string RuntimeApiKey = isLiveServer ? openAIKey : "test";
+configurable string serviceUrl = isLiveServer ? "https://api.openai.com/v1" : "http://localhost:9090";
+
+ConnectionConfig config = {
+    auth: {
+        token:RuntimeApiKey
+    }
+};
+final Client openAIAudios = check new(config,serviceUrl);
```

Comment:
```suggestion
final Client openaiAudio = check new(config,serviceUrl);
```

Reply:
@ayeshLK here shouldn't the client name be `openAIAudio`? 
The same convention is used in Ballerina AI blades as well https://ballerina.io/use-cases/ai/

Reply:
+1 to @NipunaRanasinghe's suggestion.